pkg/platform/client.go

```diff
@@ -17,22 +17,18 @@ package platform
 import (
 	"fmt"
 
+	flag "github.com/spf13/pflag"
```

Comment:
It seems strange to put flag into this platform package. Can you try to move the flags to pkg/cmd/?

Also, as you mentioned in the PR description:
"A possible break change is that platform flags have to be specified after node agent flags, otherwise the node agent flag parsing would fail."
I'm afraid this will cause trouble for our customers :( Can you please try to make the flags independent of orders? Otherwise I'm afraid we have to stick to the original approach until we find some better solution.

Reply:
Hmmm I'll try to see if I can work around this issue.

Reply:
I'll submit a separate PR for flag processing and testing, and then rebase this PR on top of that. It will allow node agent flags and platform flags mixed in order.

Reply:
Thank you for the work! :)